### PR TITLE
docs(learn): Migrate the legacy guide "Easy profiling for Node.js Applications" to the learn section 

### LINKED ISSUE #6220: Migrate the legacy guide "Easy profiling for Node.js Applications" to the learn section
Migrate the content currently at [Easy profiling for Node.js Applications](https://nodejs.org/en/guides/simple-profiling/) to a new page under the "learn" section found at https://github.com/nodejs/nodejs.org/tree/main/pages/en/learn.

- [ ] Use your judgement about what existing or new folder the content should go in. Ask in a draft PR if you are unsure. 
- [ ] Review the content during migration, please do not simply copy/paste the content. Similar to #6140, the content may need edits to account for the latest Node.js features.
- [ ] Run your content through a spell-checker in the editor, like https://github.com/streetsidesoftware/vscode-spell-checker, Word/Google Docs, Grammarly, or https://alexjs.com/

### PR DESCRIPTION
**Pull request by byt3h3ad**
## Description

Migrated the legacy guide "Easy profiling for Node.js Applications" to the learn section.

## Validation

image

## Related Issues

Fixes #6220 

### Check List

- [x] I have read the [Contributing Guidelines](https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md) and made commit messages that follow the guideline.
- [x] I have run `npx turbo lint` to ensure the code follows the style guide. And run `npx turbo lint:fix` to fix the style errors if necessary.
- [x] I have run `npx turbo format` to ensure the code follows the style guide.
- [x] I have run `npx turbo test` to check if all tests are passing.
- [ ] I've covered new added functionality with unit tests if necessary.

### CODE REVIEW

File: pages/en/learn/getting-started/simple-profiling.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Easy profiling for Node.js Applications
```
Comment (ovflowd) [inline pages/en/learn/getting-started/simple-profiling.md]: The title could be "Profiling Node.js Applications", in my opinion.

File: navigation.json [hunk 2]
```diff
@@ -124,6 +124,10 @@
             "debugging": {
               "link": "/learn/getting-started/debugging",
               "label": "components.navigation.learn.gettingStarted.links.debugging"
+            },
+            "profiling": {
+              "link": "/learn/getting-started/simple-profiling",
+              "label": "components.navigation.learn.gettingStarted.links.simpleProfiling"
```
Comment (ovflowd) [inline navigation.json]: Where's the i18n entry for this label?

**Comment by ovflowd** [comment]
I see the PR is still a draft, but don't forget to also add a redirect.json entry for this guide :)

**Comment by byt3h3ad** [comment]
hey @ovflowd i have made all the changes requested. sorry for the delay!

File: pages/en/guides/diagnostics/poor-performance/index.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+---
```
Comment (AugustinMauroy) [inline pages/en/guides/diagnostics/poor-performance/index.md]: why it's added? it's already migrate take a look to `en/learn/diagnostics/poor-performance`

**Review by ovflowd — APPROVED** [review]

**Comment by ovflowd** [comment]
Thanks for the contribution :)

**Pull request merged**